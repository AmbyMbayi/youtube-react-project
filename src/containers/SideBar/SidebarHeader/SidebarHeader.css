.sidebar__header {
    text-transform: uppercase;
    color: rgb(240, 234, 234);
    margin-top: 20px;
    margin-left: 30px;
    width: fit-content;
}