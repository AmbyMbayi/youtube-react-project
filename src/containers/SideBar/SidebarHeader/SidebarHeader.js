import React from 'react';
import './SidebarHeader.css';

function SidebarHeader(props) {
	return (
		<div>
			<p className="sidebar__header">{props.heading}</p>
		</div>
	);
}

export default SidebarHeader;
