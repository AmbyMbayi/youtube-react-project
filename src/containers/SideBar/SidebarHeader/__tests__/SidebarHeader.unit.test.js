import React from 'react';
import { shallow } from 'enzyme';
import SidebarHeader from '../SidebarHeader';

describe('SidebarHeader', () => {
	test('render empty SidebarItem', () => {
		const wrapper = shallow(<SidebarHeader />);
		expect(wrapper).toMatchSnapshot();
	});
	test('renders sidebarheadings', () => {
		const wrapper = shallow(<SidebarHeader heading="Libray" />);
		expect(wrapper).toMatchSnapshot();
	});
});
