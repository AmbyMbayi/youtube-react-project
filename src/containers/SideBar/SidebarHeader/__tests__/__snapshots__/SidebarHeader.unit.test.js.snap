// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarHeader render empty SidebarItem 1`] = `
<div>
  <p
    className="sidebar__header"
  />
</div>
`;

exports[`SidebarHeader renders sidebarheadings 1`] = `
<div>
  <p
    className="sidebar__header"
  >
    Libray
  </p>
</div>
`;
